### PR TITLE
Newfeature

### PR DESCRIPTION
**Pull request by AbednegoKibitok**
-Modify index.html
-Modify index.html
-Add my skills section
-Rectify linters errors 
-Add social media icon links
-Add my recent work
-Add README.md
-Add MIT.md
-Rectify linter errors
-Modify styles.css
-Modify README.md
-Modify index.html
-Modify styles.css

### CODE REVIEW

**Review by Gambit142 — COMMENTED** [review]
Hi @AbednegoKibitok, 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has a _**README.md**_ file where some headers are not professionally documented. Kindly do the following in the _**README.md**_ file:

- Fill out 1 - 3 features in the **_Key Features_** Section
- Fill out 1 - 3 features in the _**Future Features**_ Section
- Fill out information for the _**Install**_ and _**Usage**_ sections

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -8,29 +8,157 @@
...
+      <div>
+        <div class="htmlcss">
+          <div class="item4">css</div>
+          <div class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item6">html</div>
+          <div class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item8">bootstrap</div>
+          <div class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></div>
+          <div class="item10">ruby</div>
+        </div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure the tech stack container is wrapped into the `ul` tag and each of the is wrapped into the `li` tag in order to structure your HTML semantically correctly.

File: index.html [hunk 2]
```diff
@@ -8,29 +8,157 @@
...
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2">Multi-Post Stories<div class="bgr1"><img src="images/download__1_-removebg-preview.png" alt="Ellipse"></div></div>
+      <div class="item3">   
+        A daily selection of privately<br> personalized reads; no accounts<br>
+         or sign-ups required. has been the<br> industry's standard dummy text<br>
+          ever since the 1500s, when an<br> unknown printer took a standard<br> dummy text.
+      </div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap the paragraph into the `p` tag. :wink:

File: index.html [hunk 3]
```diff
@@ -8,29 +8,157 @@
...
-        </p>
-        <img src="images/Vector 4.png" class="line" alt="Horizontal line">
-       </section>
-   </section>
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2">Multi-Post Stories<div class="bgr1"><img src="images/download__1_-removebg-preview.png" alt="Ellipse"></div></div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap the `Multi-Post Stories` title into the heading tag `hx`. :wink:

File: index.html [hunk 4]
```diff
@@ -8,29 +8,157 @@
...
+      <li><a href="#"><img src="images/Group.png" alt="Linkedin_icon"></a></li>
+      <li><a href="#"><img src="images/VectorX.png" alt="X_icon"></a></li>
+      <li><a href="#"><img src="images/Teiitter.png" alt="Twitter_icon"></a></li>
+      <li><a href="#"><img src="images/Vectorm.png" alt="M_icon"></a></li>
+    </ul>
+    <div class="scroll">
+      <img src="images/Group 55.png" class="arrow" alt="Arrow">
+      <img src="images/Work.png" class="work" alt="work">
+    </div>
+    <div class="text">MY RECENT WORK</div>
```
Comment (levy002) [inline index.html]: - [ ] Kindly make sure to wrap the `MY RECENT WORK` title into the heading tag `hx`. :wink:

File: styles.css [hunk 5]
```diff
@@ -3,6 +3,241 @@
...
+.ellipse {
+  position: absolute;
+  padding-left: 290px;
+}
+
+.grid-container {
+  display: grid;
+  background-color: inherit;
+  padding: 10px;
+}
```
Comment (levy002) [inline styles.css]: - [ ] Kindly make sure to align each card's content using the CSS `flex-box` property as it's one of the project requirements.

**Review by levy002 — CHANGES_REQUESTED** [review]
## Hello @AbednegoKibitok   , :wave:
### I'm Levy, your PR reviewer

Good job so far :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!



## Status: REQUIRED CHANGES ♻️
_Check the comments under the review._

- [ ] Kindly make sure to wrap all of the project cards into one `section` container and make sure this main `section` container's contents is aligned on the page using `display:grid`

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @levy002 in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 6]
```diff
@@ -8,29 +8,165 @@
...
+      <li><a href="#"><img src="images/Group.png" alt="Linkedin_icon"></a></li>
+      <li><a href="#"><img src="images/VectorX.png" alt="X_icon"></a></li>
+      <li><a href="#"><img src="images/Teiitter.png" alt="Twitter_icon"></a></li>
+      <li><a href="#"><img src="images/Vectorm.png" alt="M_icon"></a></li>
+    </ul>
+    <div class="scroll">
+      <img src="images/Group 55.png" class="arrow" alt="Arrow">
+      <img src="images/Work.png" class="work" alt="work">
+    </div>
+    <div class="text"><h4>MY RECENT WORK</h4></div>
```
Comment (Zuhaib042) [inline index.html:78]: - [ ] As the previous code reviewer required to take a `section` tag here and include all the projects cards into that section container tag and aligned all the projects cards using `display: grid;`

File: index.html [hunk 7]
```diff
@@ -8,29 +8,165 @@
...
+    <img src="images/Vector 4.png" class="line" alt="Horizontal line">
+    <div class="grid-container">
+      <div class="item1"></div>
+      <div class="item2">Multi-Post Stories</div>
```
Comment (Zuhaib042) [inline index.html]: - [ ] As the previous reviewer suggested to wrap the Multi-Post Stories title into the heading tag `hx`. 
- [ ] Actually what he meant and what the requirement is to wrap all similar cards headings into `h` tag.

File: styles.css [hunk 8]
```diff
@@ -3,6 +3,242 @@
...
+  position: absolute;
+  padding-left: 290px;
+}
+
+.grid-container {
+  flex-direction: column;
+  display: grid;
+  background-color: inherit;
+  padding: 10px;
+}
```
Comment (Zuhaib042) [inline styles.css]: - [ ]  Kindly make sure to align each card's content using the CSS flex-box property as it's one of the project requirements and as it was requested by the previous code reviewer.

File: index.html [hunk 9]
```diff
@@ -8,29 +8,165 @@
...
+      <div>
+        <ul class="htmlcss">
+          <li class="item4">css</li>
+          <li class="item5"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item6">html</li>
+          <li class="item7"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item8">bootstrap</li>
+          <li class="item9"><img src="images/Rectangle 70.png" alt="Vertical line"></li>
+          <li class="item10">ruby</li>
+        </ul>
```
Comment (Zuhaib042) [inline index.html:100]: - [OPTIONAL] As one the requirement is to stick to the Figma design, I can see there are some issues in your design that I will highly recommend to fix them. and the reason of this change is mentioned below.
 - You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).
 The issues are the following:
- [OPTIONAL] In your design, the projects' cards' technologies `li`s have bullet points that are the default styling of `li`s in HTML, kindly try to remove them. there are also some mysterious black bullet points in each `li`, I will recommend to fix them as well.
- [OPTIONAL] One of the requirements in the previous milestone is to make our design responsive from 375px up to the 768px. I can see that your design is not responsive as you have given your `body` the exact width of `375px`, it made your design unresponsive. I will highly recommend to fix that. As that problem was in previous pull request `Files changed` tab. now I can't access that and I can't suggest an inline-comment in the specific place, that's why I am writing it here.
your design
qwer

figma design
qw

**Review by Zuhaib042 — CHANGES_REQUESTED** [review]
Hi @AbednegoKibitok  

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
close

 ## Highlights
✅ Linters passing
✅ Correct Github flow
You must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 10]
```diff
@@ -8,29 +8,167 @@
...
+        <a href="#" class="button">See Project</a>
+      </div>
+    </div>
+  </div>
```
Comment (IngDavidTM) [inline index.html]: - [ ] While you did a good effort wrapping the elements with `<section>` tag, as the previous code reviewer asked, please include all the projects cards into that `<section>` tag and align all the projects cards using grid :+1:

**Review by IngDavidTM — CHANGES_REQUESTED** [review]
## Hi @AbednegoKibitok  🙋‍♂️, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't been addressed yet.



## Top highlight 🎯
- The Pull Request follows the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- Linters are working properly ✅ 

## Suggested changes

- [x] _Check the comments under the review._ ⬇

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by leonardodiasb — APPROVED** [review]
# Status: ✔️ Project Approved ✔️ 

Hello @AbednegoKibitok 🔥 

You worked hard in all the instructions that the previous Code Reviewers give you👍🏽 

Now you can merge this branch 🚀 

## Highlights

 - The Pull Request follows the GitHub Flow ✅
 - The PR title and description are professional and descriptive ✅
 - Linters are working properly ✅

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**